src/sdk/bucketspace_fs.js

```diff
@@ -21,13 +41,13 @@ class BucketSpaceFS {
 
     async list_buckets() {
         try {
-            const entries = await fs.promises.readdir(this.fs_root, { withFileTypes: true });
-            const dirs_only = entries.filter(e => e.isDirectory());
+            const entries = await nb_native().fs.readdir(this.fs_config_param, this.fs_root);
+            const dirs_only = entries.filter(entree => isDirectory(entree));
             const buckets = dirs_only.map(e => ({ name: new SensitiveString(e.name) }));
             return { buckets };
         } catch (err) {
             if (err.code === 'ENOENT') {
-                console.error('nsfs: root dir not found', err);
+                console.error('BucketSpaceFS: root dir not found', err);
```

Comment:
Don't we need to use the dbg?

Reply:
I don't think so. I think that we want this error.
@guymguym WDYT?

Reply:
@liranmauda I mean instead of using 'console.error()', use dbg.error()

Reply:
Maybe, but in any case not in the scope of this PR. 
The dbg is not even required in this file.